packages/react-core/src/components/Menu/MenuItemAction.tsx

```diff
@@ -0,0 +1,72 @@
+import * as React from 'react';
+import styles from '@patternfly/react-styles/css/components/Menu/menu';
+import { css } from '@patternfly/react-styles';
+import StarIcon from '@patternfly/react-icons/dist/js/icons/star-icon';
+import { MenuContext, MenuItemContext } from './MenuContext';
+
+export interface MenuItemActionProps extends Omit<React.HTMLProps<HTMLButtonElement>, 'type' | 'ref'> {
+  /** Additional classes added to the action button */
+  className?: string;
+  /** The action icon to use */
+  icon?: 'favorites' | React.ReactNode;
```

Comment:
is there a reason the consumer should not just pass the favorites icons themselves?

Reply:
they still can, this is just for convenience

Reply:
also you can now just add `isFavorited` to the MenuItem and it will render the favorites icon

Reply:
If I am understanding the code correctly, `isFavorited` prop adds the favorites icon to the item.  To me the name implies that the Menu item has been favourited (so I expect it to appear in the Favorites group).

Reply:
yep you control whether the item has been favorited by adding the prop, initially it would be `isFavorited={false}` which is accurate

Reply:
and the top group (the group of favorited items) have `isFavorited={true}` which is also correct